openrouteservice/pom.xml

```diff
@@ -353,7 +353,7 @@
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
-            <version>2.13.3</version>
+            <version>2.15.0</version>
```

Comment:
I think it is better to keep the dependency above (log4j-1.2-api) at same version. When testing this fix locally, I still had log4j-api-2.13.3 as dependency, where I expected 2.15.0 as well.